Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -94,19 +90,17 @@ def error_handler(res):
     else:
         if res.status_code in ERROR_CODES_DICT:
             raise Exception(
-                "Your request failed with the following error: {}.\nMessage: {}".format(
-                    ERROR_CODES_DICT[res.status_code], res.text
-                )
+                f"Your request failed with the following error: {ERROR_CODES_DICT[res.status_code]}.\nMessage: {res.text}"
             )
         else:
             raise Exception(
-                "Your request failed with the following error: {}.\nMessage: {}".format(
-                    res.status_code, res.text
-                )
+                f"Your request failed with the following error: {res.status_code}.\nMessage: {res.text}"
             )
 
 
 def http_request(method, url_suffix, data=None, headers=None, resp_type='json'):
+    retries = 0 if is_time_sensitive() else 3
+    status_list_to_retry = None if is_time_sensitive() else [429]
```

Comment:
what about `timeout`, defaulting to 30s?

Reply:
the default is 15s, do we want to increase it?

Reply:
Let's decrease it when running time-sensitive-ly